va/va_enc_vp9.h

```diff
@@ -353,7 +356,20 @@ typedef struct  _VAEncPictureParameterBufferVP9
              */
             uint32_t    super_frame_flag               : 1;
 
-            uint32_t    reserved                       : 10;
+            /** \brief Indicate the block size represented by each segmentation id
+             * in Segment Map when segmentation_enabled flag is on.
+             * 0: 16x16 block size, default value;
+             * 1: 32x32 block size;
+             * 2: 64x64 block size;
+             * 3: 8x8 block size.
+             */
+            uint32_t    seg_id_block_size              : 2;
+
+            /** \brief Indicate that per segment parameters are changed.
+             */
+            uint32_t    segmentation_update_data       : 1;
```

Comment:
needs to align with Windows. Also seg map can be updated but seg data not updated. and as for Linux segmentation_update_data is always set to 1 on driver side if segmentation enabled what is not correct, because segmentation_update_data should be set to 1 only if seg parameters changed. In that case there is a difference between header parameters in output streams encoded on Windows and Linux.